openapi/access.yaml

```diff
@@ -373,10 +373,82 @@ paths:
                   value:
                     type: string
                     format: byte
-        400:
+        '400':
           $ref: '#/components/responses/400BadRequest'
-        500:
+        '500':
           $ref: '#/components/responses/500InternalServerError'
+  /status:
+    get:
+      summary: Status of the network
+      tags:
+        - Status
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Status'
+        '404':
+          $ref: '#/components/responses/404NotFound'
+        '500':
+          $ref: '#/components/responses/500InternalServerError'
+      operationId: GetNetworkStatus
```

Comment:
I would highly recommend we drop this endpoint, because block headers already contain the Chain ID so this is very redundant: https://github.com/onflow/flow-go/blob/c5a986b8588b1891c059e77656a8f831d90742aa/model/flow/header.go#L20

Reply:
You are right, I agree.

Reply:
I have removed the `/status` endpoint